projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -61,10 +80,10 @@
 @include mixins.exports('datepicker.clarity') {
   .datepicker {
     display: flex;
-    flex-direction: column;
+    flex-flow: column;
```

Comment:
Do we need that change if it's doing the same thing if not specifying the additional 'wrap' that flex-flow provides?